documentation/package.json

```diff
@@ -8,8 +8,8 @@
     "@docusaurus/preset-classic": "^2.4.0",
     "clsx": "^1.2.1",
     "prism-react-renderer": "^1.3.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
`docusaurus` will be compatible with React 18 in it's v3 version. See https://github.com/facebook/docusaurus/discussions/8469